Terminal.Gui/View/ViewDrawing.cs

```diff
@@ -80,33 +80,46 @@ public void AddRune (int col, int row, Rune ch)
         Driver.AddRune (ch);
     }
 
-    /// <summary>Clears the <see cref="Bounds"/> with the normal background color.</summary>
+    /// <summary>Clears the <see cref="Frame"/> with the normal background color.</summary>
     /// <remarks>
     ///     <para>This clears the Bounds used by this view.</para>
     /// </remarks>
-    public void Clear ()
+    public void ClearFrame ()
```

Comment:
Rename good. 👍 

But I think the XmlDoc should be a bit more explicit about it being ONLY about drawing this View and that it doesn't have anything to do with any subviews (ie it's not clearing any collections).

Reply:
```cs
    /// <summary>Clears the entire view (<see cref="Frame"/>) with the normal background color.</summary>
    /// <remarks>
    ///     <para>This method will clear the area occupied by the <see cref="Margin"/>, <see cref="Border"/>, and <see cref="Padding"/>, as
    ///     well as the <see cref="Bounds"/>. Use <see cref="Clear()"/> to clear just the <see cref="Bounds"/>.</para>
    /// </remarks>
```